### ISSUE TITLE
'false positive' on no-unknown-animations

### ISSUE THREAD

**Issue by benfrain** [stylelint/stylelint#1771]
The is a bug in the **no-unknown-animations** rule

> What CSS is needed to reproduce this issue?

e.g.

```css
@keyframes fb-FadeIn {
    0% {
        opacity: 0;
    }
    100% {
        opacity: 1;
    }
}

.fb-Test_Item {
    animation: fb-FadeIn .3s linear .7s forwards;
}
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    
    "no-unknown-animations": true
  }
}
```

> Which version of stylelint are you using?

7.1.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

no

> What did you expect to happen?

not to receive a warning

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
test.css
11:26	⚠  Unexpected unknown animation name ".3s" (no-unknown-animations) [stylelint]
11:30	⚠  Unexpected unknown animation name ".7s" (no-unknown-animations) [stylelint]
```
